modules/kernel/src/org/apache/axis2/transport/http/ApplicationXMLFormatter.java

```diff
@@ -158,6 +159,33 @@ public void writeTo(MessageContext messageContext, OMOutputFormat format,
         }
     }
 
+    private void writeXMLDeclaration(MessageContext messageContext, OutputStream outputStream, OMOutputFormat format) {
+        String xmlDeclaration;
+        String writeXmlDeclaration = (String) messageContext.getProperty("WRITE_XML_DECLARATION");
```

Comment:
Shouldn't it be better introducing new constants for WRITE_XML_DECLARATION, XML_DECLARATION_ENCODING, XML_DECLARATION_STANDALONE ?

Reply:
The above are only used once. So thought of not to have constants.

Reply:
Added